contracts/EverlongStrategy.sol

```diff
@@ -502,27 +502,49 @@ contract EverlongStrategy is BaseStrategy {
                 ""
             );
 
+            // Calculate the amount of bonds needed to reach the minimum
+            // transaction amount.
+            uint256 minimumBonds = minimumTransactionAmount().mulDivUp(
+                position.bondAmount,
+                totalPositionValue
+            );
+
+            // Calculate how many bonds out of the position need to be closed.
+            uint256 bondsNeeded;
+            if (totalPositionValue <= _targetOutput - output) {
+                // We need the entire position's bonds.
+                // We can always assume it's at least the minimum transaction
+                // amount.
+                bondsNeeded = uint256(position.bondAmount);
+            } else {
+                // `bondsNeeded <= position.bondAmount` since
+                // `_targetOutput - output < totalPositionValue`.
+                bondsNeeded = uint256(position.bondAmount).mulDivUp(
+                    (_targetOutput - output),
+                    totalPositionValue
+                );
+
+                // We need to close at least the `minimumTransactionAmount`
+                // value of bonds. It's safe to assume that the position
+                // has at least that much.
+                bondsNeeded = bondsNeeded.max(minimumBonds);
```

Comment:
see above: shouldn't this just be `_poolConfig.minimumTransactionAmount`?

Reply:
After applying changes from above I decided to leave `minimumBonds` as a variable with a comment on why conversion isn't necessary.

I know it's a bit more gas but I feel like it's worth it for the readability/explanation.

Happy to change if you think otherwise :)